test_conformance/subgroups/subgroup_common_templates.h

```diff
@@ -394,6 +396,22 @@ void genrand(Ty *x, Ty *t, cl_int *m, int ns, int nw, int ng)
 {
     int nj = (nw + ns - 1) / ns;
 
+    // limit possible input values to avoid arithmetic rouding/overflow issues.
+    // for each subgroup for 4 workitems set 4 values different than 1
+    // for rest of workitems set 1
+    // shuffle values
+    // below values are 0.5f, 0.25f, 2.0f, 4.0f
+    std::vector<cl_ulong> safe_half_values{ 0x3800, 0x3400, 0x4000, 0x4400 };
+    if (std::is_same<Ty, subgroups::cl_half>::value)
+    {
+        std::random_device random_device;
+        std::mt19937 mersenne_twister_engine(random_device());
```

Comment:
In my experience we generally avoid randomizing seeds for testing, and never without capturing and reporting the seed value so it can be reproduced. Is there a reason we need it here?

Reply:
I've set constant seed - see latest